panda/src/putil/bamEnums.h

```diff
@@ -69,6 +69,12 @@ class EXPCL_PANDA_PUTIL BamEnums {
     BTM_basename,
     BTM_rawdata
   };
+
+  // Level of compression for textures in the .bam
```

Comment:
Is this truly a compression level, or just the compression format/algorithm?

Reply:
In my opinion, the enums should both indicate the compression algo and the compression level in the future e.g. `BTC_zlib_5`. But you are right, the comment is misleading.

Reply:
Can you add a space before the `{` here, to adhere to the current code style?